packages/expo-font/plugin/src/withFontsAndroid.ts

```diff
@@ -26,3 +36,38 @@ export const withFontsAndroid: ConfigPlugin<string[]> = (config, fonts) => {
     },
   ]);
 };
+
+export const withXmlFontsAndroid: ConfigPlugin<XmlFonts[]> = (config, fonts) => {
+  return withMainApplication(config, async (config) => {
+    for (const { fontName, fontFiles } of fonts) {
+      const xmlFileName = fontName?.toLowerCase().replace(/ /g, '_')!;
+      const resolvedFonts = await resolveFontPaths(fontFiles!, config.modRequest.projectRoot);
+      const fontXml = generateFontFamilyXml(resolvedFonts);
+      const fontsDir = path.join(config.modRequest.platformProjectRoot, 'app/src/main/res/font');
+      await fs.mkdir(fontsDir, { recursive: true });
+      const xmlPath = path.join(fontsDir, `${xmlFileName}.xml`);
+      await fs.writeFile(xmlPath, fontXml);
+
+      await Promise.all(
```

Comment:
the for loop runs serially but could use the same construct as here to be more efficient